Src/AutoFixture.NUnit3/InlineAutoDataAttribute.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework.Interfaces;
+
+namespace Ploeh.AutoFixture.NUnit3
+{
+    /// <summary>
+    /// This attribute acts as a TestCaseAttribute but allow incomplete parameter values, 
+    /// which will be provided by AutoFixture. 
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+    [CLSCompliant(false)]
+    public class InlineAutoDataAttribute : AutoDataAttribute
```

Comment:
In the other Glue Libraries, InlineAutoDataAttribute *is-a* CompositeDataAttribute.

It's just that CompositeDataAttribute isn't implemented yet, and so InlineAutoDataAttribute for the time being may just be an AutoDataAttribute (the base of all auto data-theory attributes).

Once the CompositeDataAttribute is available, I could imagine that this proposed implementation could be refactored and simplified...